demo/simple/move-first-child.html

```diff
@@ -0,0 +1,50 @@
+<head>
+  <title>Virtual Scoller Demo - Moving children</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+  <style>
+    .scroller {
+        overflow: scroll;
+        height: 400px;
+    }
+  </style>
+</head>
+
+<body>
+  <p>Repeatedly moving the first child to the end of the content. This
+  is potentially faster as we may be able to avoid layout changes to
+  the locked elements, however the scroller's internal state must
+  react to the mutation.</p>
+
+  <script type="module">
+    // Do this separately. If virtual-scroller is not available, the demo
+    // will still work.
+    import 'std:elements/virtual-scroller';
+  </script>
+  <script type="module">
```

Comment:
All of these could be a bit nicer by just doing `<script type="module" src="..."></script>`

Reply:
However you may want to just have a single module that's something like demo-scaffolding.mjs which:

- Imports all three of these
- Adds a confirm-features element
- Adds a fps-display element
- Adds a demo-controller element
- Dynamic imports `import("std:elements-virtual-scroller")` (dynamic import failure won't prevent subsequent code from executing)

That could really DRY up the demos.

Reply:
Done. Didn't know about dynamic import.